larray/inout/xw_excel.py

```diff
@@ -661,6 +664,10 @@ def open_excel(filepath=None, overwrite_file=False, visible=None, silent=None, a
 >>> wb['arr3'] = arr3.dump()                                      # doctest: +SKIP
 >>> wb.save()                                                     # doctest: +SKIP
 >>> wb.close()                                                    # doctest: +SKIP
+"""
+
+    Workbook.app.__doc__ = """
+Return the currently active Excel instance.
```

Comment:
This is wrong/misleading. It should rather be something along the lines of: `Return the Excel instance used by this workbook` or  `Return the Excel instance this workbook is attached to`.